telegram/_bot.py

```diff
@@ -1356,7 +1355,12 @@ async def send_sticker(
             chat_id (:obj:`int` | :obj:`str`): |chat_id_channel|
             sticker (:obj:`str` | :term:`file object` | :obj:`bytes` | :class:`pathlib.Path` | \
                 :class:`telegram.Sticker`): Sticker to send.
-                |fileinput|
+                Pass a ``file_id`` as string to send a sticker that exists on the Telegram
+                servers (recommended), pass an HTTP URL as a string for Telegram to get a
+                ``".WEBP"`` sticker from the Internet, or upload a new ``".WEBP"`` or ``".TGS"``
+                sticker. Video stickers can only be sent by a ``file_id``. Animated stickers can't
+                be sent via an HTTP URL.
+
```

Comment:
```suggestion
                |fileinput| Video stickers can only be sent by a ``file_id``. Animated stickers can't
                be sent via an HTTP URL.

```

Reply:
No, it's different, this one mentions webp sticker from internet, or upload webp/tgs sticker.

Reply:
The additional

> Video stickers can only be sent by a ``file_id``. Animated stickers can't be sent via an HTTP URL.

Should already clarify that IMO ...